docs/core/04-contracts-overview.md

```diff
@@ -0,0 +1,117 @@
+
+### Framework Infrastructure
+
+#### DAO Creation
+
+| Contract      | Description                                                | Relationship    |
+| :------------ | :--------------------------------------------------------- | :-------------- |
+| `DAOFactory`  | A template to setup plugins for a DAO using our framework. | creates `DAO`   |
+| `DAORegistry` | Registers DAOs                                             | registers `DAO` |
+
+#### Plugin Management
+
+| Contract             | Description                                                         | Relationship                                         |
+| :------------------- | :------------------------------------------------------------------ | :--------------------------------------------------- |
+| `PluginSetup`        | A template to setup plugins for a DAO using our framework.          | `Plugin`, `PluginCloneable`, `PluginUUPSUpgradeable` |
+| `PluginRepo`         | Versions a plugin by storing the associated `PluginSetup` contract. | versions `PluginSetup`                               |
+| `PluginRepoFactory`  | Creates `PluginRepo` contracts                                      | creates `PluginRepo`                                 |
+| `PluginRepoRegistry` | Registers `PluginRepo` contracts                                    | registers `PluginRepo`                               |
+
+### Plugin Implementations
+
+| Contract             | Description                                                                                  | Relationship                                                                    |
+| :------------------- | :------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------ |
+| `MerkleMinter`       | Mints `ERC-20` tokens and distributes them on merkle trees using `MerkleDistributor` clones. | is `PluginUUPSUpgradeable`, uses `MerkleDistributor`, is used by `TokenFactory` |
+| `MerkleDistributor`  | Distributes claimable ERC-20 tokens via a merkle tree.                                       | is `PluginUUPSUpgradeable`                                                      |
+| `MajorityVotingBase` | Contains abstract, majority voting functionality.                                            | is parent of `ERC20Voting` and `AllowlistVoting`                                |
+| `ERC20Voting`        | A majority voting plugin using ERC-20 tokens for the census.                                 | inherits from `MajorityVotingBase`                                              |
+| `AllowlistVoting`    | A majority voting plugin using a list of allowed addresses for the census.                   | inherits from `MajorityVotingBase`                                              |
```

Comment:
I would start with these 2 first, then show the rest in a separate table

Reply:
Amended